views/css/style.css

```diff
@@ -1,9 +1,27 @@
 body {
     color: #252D4F;
 }
+
+/* ---- Halloween Scrollbar Styling */
+
+.halloween::-webkit-scrollbar
+{
```

Comment:
Split out halloween and christmas styles into their own css files and so only load them when needed.

Reply:
Done